src/node_api.cc

```diff
@@ -1149,6 +1149,25 @@ napi_status napi_create_array_with_length(napi_env e,
   return GET_RETURN_STATUS();
 }
 
+napi_status napi_create_string_latin1(napi_env e,
+                                      const char* s,
```

Comment:
e and s should be full words, for example e should b env.  There is another PR fixing this up across the napi codebase so we should make sure we keep it up.